libs/exec/execv_windows.go

```diff
@@ -35,7 +38,12 @@ func execv(opts ExecvOptions) error {
 
 	err = cmd.Wait()
 	if exitErr, ok := err.(*exec.ExitError); ok {
-		os.Exit(exitErr.ExitCode())
+		// Cleanup the temporary script since we directly exit here.
+		if opts.windowsCleanup != nil {
+			opts.windowsCleanup()
+		}
+		opts.windowsExit(exitErr.ExitCode())
```

Comment:
You can refactor this out into separate `windowsExit` function which you can reuse so you won't forget to call opts.windowsCleanup

Reply:
Thanks, done.